python/ray/workflow/serialization.py

```diff
@@ -113,53 +109,57 @@ async def export_stats(self) -> Dict[str, Any]:
 OBJECTS_DIR = "objects"
 
 
-def obj_id_to_paths(workflow_id: str, object_id: str) -> List[str]:
-    return [workflow_id, OBJECTS_DIR, object_id]
+def _obj_id_to_key(object_id: str) -> str:
+    return os.path.join(OBJECTS_DIR, object_id)
```

Comment:
I'm thinking about s3 and windows. Will this still work? I mean for s3, it's like 'a/b/c' and for windows, it's like 'a\b\c'

Reply:
it should still works. the storage would turn `/` into `\` with `.resolve()`